pkgs/development/haskell-modules/patches/inline-c-cpp-fcompact-unwind.patch

```diff
@@ -0,0 +1,13 @@
+--- a/inline-c-cpp.cabal
++++ b/inline-c-cpp.cabal
+@@ -47,6 +47,10 @@ common cxx-opts
+     ghc-options:
+       -optcxx-std=c++11
+       -optcxx-Wall
++
++    if os(darwin)
```

Comment:
I think it's simpler to just add an `appendConfigureFlags [ "--ghc-option=-fcompact-unwind" ]` to `configuration-darwin.nix`?

This could be upstreamed, but with a lower bound on GHC 9.2.2 as it stands of course.

Reply:
I've linked the upstream `inline-c-cpp` pr in comments (https://github.com/fpco/inline-c/pull/131).

The overrides are now all in the version-specific files because each one has a different context, which was hard to understand in a single comment.